source/extensions/filters/network/zookeeper_proxy/decoder.cc

```diff
@@ -64,6 +65,7 @@ void DecoderImpl::decodeOnData(Buffer::Instance& data, uint64_t& offset) {
   //       However, some client implementations might expose setWatches
   //       as a regular data request, so we support that as well.
   const int32_t xid = helper_.peekInt32(data, offset);
+  ENVOY_LOG(trace, "zookeeper_proxy: decoding request with xid {} at offset {}", xid, offset);
```

Comment:
Let merge line 46, 51 into this log to avoid too many log

Reply:
I did not merge L51 into L68 since there are length checks at L52-53.